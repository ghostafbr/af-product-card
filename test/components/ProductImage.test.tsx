import * as React from 'react';
import renderer, {ReactTestRenderer} from 'react-test-renderer';
import {ProductImage} from '../../src';
import {product2} from '../data/products';

describe('ProductImage', () => {

    it('should display the component with the correct personalized image', (): void => {
        const wrapper: ReactTestRenderer = renderer.create(<ProductImage img="https://hola.jpg" className="custom-image" />);
        expect(wrapper.toJSON()).toMatchSnapshot();
    });

    it('should display the component with the name of the product', (): void => {
        const wrapper: ReactTestRenderer = renderer.create(<ProductImage img={product2.img} className="custom-image" />);
        expect(wrapper.toJSON()).toMatchSnapshot();

    });

});
