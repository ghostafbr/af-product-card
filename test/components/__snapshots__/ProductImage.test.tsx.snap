// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage should display the component with the correct personalized image 1`] = `
<img
  alt="Product"
  className="productImg custom-image"
  src="https://hola.jpg"
/>
`;

exports[`ProductImage should display the component with the name of the product 1`] = `
<img
  alt="Product"
  className="productImg custom-image"
  src="./coffee-mug.png"
/>
`;
